### ISSUE TITLE
Remove Gamma user access to OARS data

### ISSUE THREAD

**Issue by pomegranited** [openedx/tutor-contrib-oars#15]
By default, Open edX SSO users are assigned the Gamma role, which indicates that they are not course authors, staff or superusers.
However, the row-level security filters are currently linked to the Open edX role (not the Gamma role), and so they only apply to that role.

These RLSFs need to be fixed to apply to all SSO users, regardless of role.

**Comment by Ian2012** [comment]
@pomegranited Could you please leave more details?

I tried changing the default role to [Open edX](https://github.com/openedx/tutor-contrib-superset/blob/main/tutorsuperset/templates/superset/apps/pythonpath/superset_config_docker.py#L60) but that role doesn't have any permissions at all.

Should I add the minimum permission configuration that allows Open edX role to use superset?

**Comment by pomegranited** [comment]
@Ian2012 

> Should I add the minimum permission configuration that allows Open edX role to use superset?

Yes. This task could be rephrased as "Replace 'Gamma' role with 'Open edX' role", so that:

* 'Gamma' users (if created) have no access to any Open edX data
* 'Open edX' users have access to Open edX data as allowed by their course staff roles.

All new SSO users should be created with role 'Open edX'.

**Comment by Ian2012** [comment]
assign me

**Comment by Ian2012** [comment]
@bmtcril this one should be closed

**Comment by bmtcril** [comment]
👍
